pkg/util/net/http.go

```diff
@@ -67,6 +70,9 @@ func SetTransportDefaults(t *http.Transport) *http.Transport {
 	if t.TLSHandshakeTimeout == 0 {
 		t.TLSHandshakeTimeout = defaultTransport.TLSHandshakeTimeout
 	}
+	if err := http2.ConfigureTransport(t); err != nil {
```

Comment:
I'm confused. Doesn't 1.6.2 support http2 seamlessly? Why do we have to do this?